lib/generators/avo/templates/initializer/avo.tt

```diff
@@ -100,6 +100,9 @@ Avo.configure do |config|
   #   file_logger
   # }
 
+  ## == Audit ==
+  # config.audit = false
```

Comment:
Let's think about the configuration of this gem a bit more. true/false is a bit terse and we'll for sure have more configs on that gem.
How about moving this config as `config.enabled = true|false` on that gem's configuration?

Reply:
>How about moving this config as `config.enabled = true|false` on that gem's configuration?

Well, that's a really good idea to have a configuration file for each gem!